core.js

```diff
@@ -1,14 +1,14 @@
 module.exports = {
-  parser: '@babel/eslint-parser',
+  parser: '@typescript-eslint/parser',
   parserOptions: {
     sourceType: 'module',
-    requireConfigFile: false,
+    requireConfigFile: false
   },
   env: {
     browser: true,
     node: true,
     es2020: true,
-    jest: true,
```

Comment:
where are our trailing commas going, do you not have prettier set up for your vscode or do we not have a prettier config in this project?